### ISSUE TITLE
Blog Plan and Rota

### ISSUE THREAD

**Issue by ghost** [dwyl/blog#19]
I suggest we kick off the blog with a rota of the full-timers releasing one new piece of content a week every Friday. Posts can be one of the ideas already in the repo or anything someone wants to write about. Have added some suggestions in case people struggle to pick.

- [x] Post 1 - 29 Sep - Mark - #1
- [ ] Post 2 - 6 Oct - Inês - [Suggestions: #2 #9 #11 #14 #15]
- [ ] Post 3 - 13 Oct - Nelson [Suggestions: #5 #6]
- [ ] Post 4 - 20 Oct - Naaz [Suggestion: #17]
- [ ] Post 5 - 27 Oct - Simon [Suggestion: #18 - could be written in French(?)]
- [ ] Post 6 - 3 Nov - Cleo [Suggestion: #16]

...*repeats*

Because full-timer contracts are 8h and client days are 7h, this gives us significant time to write them 1h (per day) x 5 (week) x 6 (rota) = 30h for miscellaneous dwyl tasks inc posts

Some posts can be written in a very short time though, so however long it takes depends on what you want to write.

**Comment by markwilliamfirth** [comment]
@iteles are you able to post something today or should we rearrange the schedule?

**Comment by markwilliamfirth** [comment]
@iteles @nelsonic I haven't heard back from you guys on this yet - do you want to rearrange the posting schedule?

**Comment by iteles** [comment]
Apologies for taking so long to reply here. I had a lot of questions when I first looked at this and was seeing if anyone else might ask them in case it was just me! I also got waylaid by #20

I will get a realistic schedule to you (for my time) as soon as possible. 

With regards to others blogging, this is fantastic and I would like to see it happen. 
As you know, for the most part, we use that last hour of the day for QAing and scrum mastering across other projects (the reasoning around this and a mini study of whether it works is probably a good blog topic in and of itself), so it's not a 'free' hour for everyone right now. We just need a little more thought as to how to make this happen. 

It would be good to provide some information to people _ahead_ of time as to _why_ having a blog is important and why they should even be contributing. A 'Why?' and a 'What?' section in the readme would suffice for this. 

Is there anything else people should know before writing their blog? I don't mean the super dry, 'here's a page of legal mumbo jumbo to read' stuff. Is there a tone we would like them to follow? A word limit? Restrictions on confidential information regarding our clients? What is the audience for the blog post? What is the expected outcome? Any support available to help them turn a random topic from a 'this is what I had for breakfast' type post into something useful that people will look to read? Any support on forming a good title?

Logistically, how do people submit blog drafts? Who does it get submitted to? What's the process for getting it on the website?

In addition, have we done any research on common pitfalls for company blogs? Can we avoid some pain down the line by doing  an hour of research into this before we start?

**Comment by markwilliamfirth** [comment]
There has been talk on GitHub about having a blog [since 6 June 2015](https://github.com/dwyl/dwyl-site/issues/5). There are also mentions of suggested blog posts on Trello from as early as [29 Aug 2014](https://trello.com/c/egy45kI1/100-blog-vagrant). This is something you and Nelson have been talking about for years, so I would have thought you would have been keen to get writing as soon as possible.

The above questions suggest that it's unclear who owns this, so please can you clarify that here: https://github.com/dwyl/hq/issues/324

With respect to the points you raise, I don't think these are blockers and I don't think these are major issues right now. These are things we can work out a process for as we do it. I think the major issue is needing to **post regular content on a schedule**. We can revisit and edit and/or even delete posts in future.

Regardless I will answer your questions below:

why having a blog is important
> I think this one should be answered by you and/or @nelsonic, as it was your/his idea

why FTs should even be contributing
> Many reasons, mainly so that the blog has diversity in its content that reflects the diversity of the organisation. I don't see the blog as being just FTs, it should expand to anyone in the dwyl community (and even outside of it - guest posts) who wishes to post, but for now we just need to get some posts up to get it started

Is there a tone we would like them to follow?
> No, we can iterate and decide on this later by testing and seeing what works

A word limit?
> No, we can iterate and decide on this later by testing and seeing what works

Restrictions on confidential information regarding our clients?
> Yes, all FTs have confidentiality clauses in their contracts

What is the audience for the blog post?
> Currently, people online wanting to learn more about dwyl and what we do and what we're about

What is the expected outcome?
> To have an initial set of blog posts where people can read about dwyl and some of the things we do and our thoughts on topics, so that the blog isn't bare

Any support available to help them turn a random topic from a 'this is what I had for breakfast' type post into something useful that people will look to read? Any support on forming a good title?
> People can create an issue in this rep if they need support with something

Logistically, how do people submit blog drafts? Who does it get submitted to? What's the process for getting it on the website?
> Post drafts can be written up in documents, emails or posted in issues (however people prefer) before being PR-ed to the dwyl-site repo

In addition, have we done any research on common pitfalls for company blogs? Can we avoid some pain down the line by doing an hour of research into this before we start?
> There are no major pitfalls associated with this initial set of posts, as all content goes through a review stage and we can iterate and edit them at a later date if required. The major pitfall here is not posting.

**Comment by iteles** [comment]
Mark. I have not said that this is not important and I don't dispute that we have been discussing it for a long time. This is why I said I would get back to this issue with a realistic schedule for my posts asap. 

On your other notes:
+ Seeing as your current job description includes marketing and PR (in line with dwyl's values and beliefs, as ever), including the website, and given that you opened this issue, I thought that it was clear that this was in your purview, my apologies if not
+ I don't feel the need to reply to the rest of your comment point by point, nor was I implying it was a blocker, I was simply pointing out that you are suggesting dwylers do something that may be outside of their comfort zone without providing _any_ parameters, which doesn't work for everyone. As such, I was merely noting some of the questions that _may_ come up and that it might be nice for people to know some thought has gone into this rather than it being 'everyone is doing it so we're making you do it too'

**Comment by markwilliamfirth** [comment]
Seeing as your current job description includes marketing and PR (in line with dwyl's values and beliefs, as ever), including the website, and given that you opened this issue, I thought that it was clear that this was in your purview, my apologies if not
> This is still unclear - https://github.com/dwyl/hq/issues/324 is still an open issue

**Comment by markwilliamfirth** [comment]
@iteles do you have a realistic schedule for posts now?

**Comment by markwilliamfirth** [comment]
@iteles it's now been 3 weeks - is there any update on this one?

**Comment by markwilliamfirth** [comment]
@iteles it's now been 3 weeks - is there any update on this one?

**Comment by iteles** [comment]
Closing this as there's a lot of (minimum effort possible) noise and no longer relevant.
